host_vars/mongodb.opencraft.hosting/public.yml

```diff
@@ -0,0 +1,11 @@
+mongodb_user_admin_name: "{{ MONGODB_USER_ADMIN_NAME }}"
+mongodb_user_admin_password: "{{ MONGODB_USER_ADMIN_PASSWORD }}"
+
+mongodb_root_admin_name: "{{ MONGODB_ROOT_ADMIN_NAME }}"
+mongodb_root_admin_password: "{{ MONGODB_ROOT_ADMIN_PASSWORD }}"
+
+mongodb_keyfile_content: "{{ MONGODB_KEYFILE_CONTENT }}"
+
+TARSNAP_KEY: "{{ MONGODB_TARSNAP_KEY }}"
+SANITY_CHECK_SNITCH: "{{ MONGODB_SANITY_CHECK_SNITCH }}"
+TARSNAP_BACKUP_SNITCH: "{{ MONGODB_TARSNAP_BACKUP_SNITCH }}"
```

Comment:
@antoviaque @smarnach This is not the shining point of Ansible. Ansible variable precedence is [defined here](http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable), but I here is the relevant part (lower entries have higher precedence: 

- inventory group_vars
- inventory host_vars
- playbook group_vars
- playbook host_vars
- ( ... )
- extra vars (always win precedence)

For now public repositories contain variables that fall into 'playbook'  category, and private variables fall into `inventory category`. So if we started using `host_vars` in secret repository `group_vars` from public repository would have precedence, which is IMO non-intuitive, and could be error prone (as one naturally assumes that `host_vars` override `group_vars`). Private vars have nice property of having biggest priority. .

On the other hand having one big `private-vars` (that is not easily mergeable) is also a pain. 

Cleanest solution would be to move all `host_vars` and `group_vars` to secret repository, but this would make everything less useful to others (which could be fixed by having updated documentation).